src/client/containers/GamePage/GamePage.component.js

```diff
@@ -1,4 +1,34 @@
 import React from 'react';
-import HeaderTitle from '../../components/Title/Title';
+import Footer from '../../components/FooterComponent/FooterComponent';
+import Map from '../../components/CircleAndMarkerCurrentLocation/MarkerCurrentLocation';
+import Scores from '../../components/AchievementComponent/Scores';
+import keyImage from '../../assets/images/icons/kye-icon.svg';
+import timerImage from '../../assets/images/icons/timer-icon.svg';
+import cupImage from '../../assets/images/icons/kup-icon.svg';
 
-export const GamePage = () => <HeaderTitle title="This is the Game Page" />;
+const divStyle = {
+  display: 'flex',
+  justifyContent: 'space-around',
+  alignItems: 'center',
+  width: '100%',
+  position: 'fixed',
+  top: '35px',
+  zIndex: '100',
+  color: '#ccc',
+};
```

Comment:
Why are you using CSS-in-JS here? Please put CSS in CSS files unless you need to dynamically update the CSS via js.